README.md

```diff
@@ -1 +1,169 @@
-# Hello_Microverse-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microverse-] <a name="about-project"></a>
+
+**[Hello_Microverse-]** is a first project in Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+I'm using mainly html, css for building this project.
+
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_Page]**
+

```

Comment:
- [ ] you did a great effort 😃 at adding the key features section as you know the importance of the key features to describe the advantages of the project and the features you used in it, so please try to make the key features more descriptive to describe the features in the project, so let's make it professional by saying. i.e. below and with that your key features will be great😊💯 💯🚀

```e.i

- Set up the linter for the project for writing clean code👌
- Add the test file and node modules to the .gitignore file🚀
- Add the styling color using CSS

```